### ISSUE TITLE
Issue with long duration motion 

### ISSUE THREAD

**Issue by longchen-geo** [LuisMontejo/REQPY#1]
Hi, thanks for developing this script, it has been very helpful. I want to raise your awareness that I recently noticed an issue when doing spectral matching on motions with long period contents, i.e., subduction motions; the modified motion can sometimes truncate the long period contents and result in very different displacement time history comparing to the original motion. The attached figure shows the issue. This motion is one of the Tohoku records and is almost 300 seconds long. In this case, I set the target spectrum to be the same as the motion's original response spectrum (matching to itself) and TL2 = 10s. As you can see in the figure, REQPY works great in terms of acceleration, but mismatched the velocity and displacement due to truncating of the low frequency content. 

![image](https://user-images.githubusercontent.com/16639187/180892633-9f1b533c-503a-448e-9861-4675bf0a9704.png)

To avoid this issue, the user may need to specify the spectral matching period window to long period, 100 seconds in some extreme cases. The figure below shows the results with TL2 = 100s. The matched motion was greatly improved. 

![image](https://user-images.githubusercontent.com/16639187/180893332-a2d9b3fc-09c3-4763-9dce-5639f455282b.png)

**Comment by LuisMontejo** [comment]
Hi! thanks for noticing this. Would you mind sharing the ground motion file. I would like to address this directly in the code and use the same motion for validation.

**Comment by longchen-geo** [comment]
[KNG007_NS_X.txt](https://github.com/LuisMontejo/REQPY/files/9203882/KNG007_NS_X.txt)
[KNG007_EW_Y.txt](https://github.com/LuisMontejo/REQPY/files/9203883/KNG007_EW_Y.txt)
[targetSpectrum.txt](https://github.com/LuisMontejo/REQPY/files/9203885/targetSpectrum.txt)

**Comment by LuisMontejo** [comment]
Thanks, 
 - the results you are showing are for single-component match or simulatneous match of both components to a target RotD100?

**Comment by longchen-geo** [comment]
That's matching both components to the target RotD100 spectrum. The plotted time history was just showing one of the two components, but they both show the same pattern.

**Comment by LuisMontejo** [comment]
I modified the definition of the low bound of the frequency range for decomposition (FF1), it is now defined based on the duration of the record to account for low frequency contents on long duration records. Below are the results. 
I have also added the code for this example in the documentation (example 4) and updated the REQPY_module.

![image](https://user-images.githubusercontent.com/31960642/181635256-b2e28673-1254-4e32-a0b3-18d4d91f1e49.png)
![image](https://user-images.githubusercontent.com/31960642/181635369-429a519e-2cef-43be-90c3-66610b7c983a.png)
